pkgs/by-name/li/libnvidia-container/package.nix

```diff
@@ -85,11 +93,25 @@ stdenv.mkDerivation rec {
   '';
 
   env.NIX_CFLAGS_COMPILE = toString [ "-I${libtirpc.dev}/include/tirpc" ];
-  NIX_LDFLAGS = [ "-L${libtirpc.dev}/lib" "-ltirpc" ];
+  NIX_LDFLAGS = [
+    "-L${libtirpc.dev}/lib"
```

Comment:
Another note on legacy, this can be probably removed:

```
❯ nix build nixpkgs#libtirpc.dev
❯ find result-dev/lib
result-dev/lib
result-dev/lib/pkgconfig
result-dev/lib/pkgconfig/libtirpc.pc
```

Makes sense since we don't usually put libraries in `$dev`.

Reply:
Ok. Looking at the code, it seems that the RPC features don't work without libtirpc at all (so they probably didn't work for quite some time). I just include the right library paths now